OrbitQt/orbitlivefunctions.cpp

```diff
@@ -103,3 +103,14 @@ void OrbitLiveFunctions::AddIterator(size_t id, Function* function) {
 QLineEdit* OrbitLiveFunctions::GetFilterLineEdit() {
   return ui->data_view_panel_->GetFilterLineEdit();
 }
+
+void OrbitLiveFunctions::Reset() {
+  live_functions_.Reset();
+
+  for (auto& [_, iterator_ui] : iterator_uis) {
```

Comment:
mind blown :-)

Reply:
I believe that its name is "Structured Binding", a C++17 feature that makes your daily life more confortable :)